gulpfile.ts

```diff
@@ -73,3 +74,27 @@ gulp.task(
 gulp.task('default', gulp.series('cjs', 'esm', 'umd', 'umd:min'));
 
 gulp.task('isolate', gulp.series(createIsolateTasksFromConfig(isolateBuildConfig)));
+
+const project: { ref?: gulpTs.Project } = { };
+
+gulp.task(
+  'dev',
+  gulp.series(
+    function build() {
+      return gulpFactory(
+        { globs: 'src/**/*.ts' },
+        'dist/dev',
+        {
+          module: 'ES2015',
+          declaration: true
+        },
+        true,
+        false,
+        project
+      );
+    },
+    function watch() {
+      return gulp.watch('src/**/*.ts', gulp.series('dev'));
```

Comment:
Does `gulp` have some cache when do hot update?

It seems to re-build all the things.

Reply:
We are already doing an incremental build, but we cannot build an incremental build for babel and terser.

So, to improve the hot reload speed, the babel and terser must be removed from dev mode.